src/js/subtraction/components/Attribution.js

```diff
@@ -0,0 +1,11 @@
+import { Attribution, NameAttribution } from "../../base";
+import React from "react";
+
+export function SubtractionAttribution({ handle, time }) {
+    if (handle && time) {
+        return <Attribution user={handle} time={time} />;
+    } else if (handle) {
+        return <NameAttribution user={handle} time={time} />;
+    }
```

Comment:
```suggestion
    if (handle) {
            if (time) {
                return <Attribution user={handle} time={time} />;
            }
            
            return <NameAttribution user={handle} time={time} />;
    }
```
Also, `NameAttribution` doesn't take a `time` prop.

Reply:
Nice, small efficiency bump, nice catch. Also yes on the same size thing. I was worried the `InitialIcon` might change the size but the box model reports that either way the flexbox is 42.85 pixels tall. I could probably also hardcode it, but I can't say that I see a point.

Reply:
Updated the look of the no data available below:
![image](https://user-images.githubusercontent.com/59776400/146618748-3584ff7e-9b2b-4116-8061-a2c316875d50.png)